ext/benchmark_utils.jl

```diff
@@ -68,8 +69,11 @@ function get_trial(f, args, name; device, with_cu_prof = :bprofile, trace = fals
         if crop
             println(p) # crops by default
         else
-            io = IOContext(stdout, :crop => :horizontal)
-            show(io, p)
+            envs = isnothing(hcrop) ? () : ("COLUMNS" => hcrop,)
+            withenv(envs...) do
+                io = IOContext(stdout, :crop => :horizontal, :limit => true, :displaysize => displaysize())
+                show(io, p)
+            end
```

Comment:
I don't know if support for Windows is something we care about, but this might not work on Windows.

Maybe you also want to add a comment or a link to your GitHub comment to explain this piece of code.

Reply:
Why would it not work on Windows?

Reply:
I'll merge for now and see how it goes

Reply:
Ah, interesting. It's probably not a big deal for us, when I run locally I usually just crop or don't crop, and it's usually pretty easy to re-run it in the same session